src/Tests/Microsoft.NET.Sdk.BlazorWebAssembly.Tests/GenerateBlazorBootJsonTest.cs

```diff
@@ -91,7 +92,7 @@ public void GroupsResourcesByType()
                         ("FileHash", "my-custom-extensionhash"),
                         ("RelativePath", "my-custom-extension.blz"),
                         ("TargetPath", "_bin/my-custom-extension.blz"),
-                        ("AssetTraitName", "BlazorWebAssemblyResource"),
+                        ("AssetTraitName", "WasmResource"),
```

Comment:
We need to be careful here because I believe we use this on the service worker. Just making a note here so that I don't forget as I haven't gone through the entire PR